### ISSUE TITLE
Structures generating beneath islands

### ISSUE THREAD

**Issue by rotorhub** [devs-immortal/The-Aether#162]
Idk if its an issue with just me but vanilla and modded structures are generating beneath the floating islands in the Aether dimension

**Comment by MBatt1** [comment]
Do you have any screenshots you can post?

**Comment by rotorhub** [comment]
![image](https://user-images.githubusercontent.com/28743246/113918330-49047800-97b0-11eb-84fe-2e32a084fc0a.png)

![image](https://user-images.githubusercontent.com/28743246/113918477-7b15da00-97b0-11eb-9d4a-e82a23209b41.png)

If its not an issue with your mod then it may be an issue with Repurposed Structures (https://www.curseforge.com/minecraft/mc-mods/repurposed-structures-fabric?page=3)

**Comment by rotorhub** [comment]
Oh also the biome names appear all janky like that in minimap mods, another mod called "Vanilla+ Biomes" has this same issue

**Comment by HerrahStrings** [comment]
This is a non-issue in repurposed structures, which offers a config option to blacklist dimensions (Just add "the_aether:the_aether") - If i understand correctly, the villages are spawning "on top of" the portal blocks at y=0. It's that mod author's discretion on whether they want to take any other tradeoff compatibility steps.

Mo'structures also had this issue, but it was fixed in a patch I requested a few days ago.
